IdentityCore/src/broker_operation/request/MSIDBaseBrokerOperationRequest.h

```diff
@@ -0,0 +1,19 @@
+//
+//  MSIDBaseBrokerOperationRequest.h
+//  IdentityCore iOS
+//
+//  Created by Rohit Narula on 2/6/20.
+//  Copyright © 2020 Microsoft. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MSIDBaseBrokerOperationRequest : NSObject
```

Comment:
Could this be a protocol? Doesn't seem like this class is doing much.

Reply:
As discussed if we make this a protocol then we will need to do a lot of casting in methods where we register operation for request and other places.